PSKoans/Koans/dbatools/AboutDbaDatabase.ps1

```diff
@@ -0,0 +1,63 @@
+#requires -modules dbatools
+using module PSKoans
+[Koan(Position = 1001)]
+param()
+<#
+    Get-DbaDatabase
+
+    The Get-DbaDatabase command gets SQL database information for each database
+    that is present on the target instance(s) of SQL Server by default. If the 
+    name of the database is provided, the command will return only the specific 
+    database.
+#>
+Describe 'Get-DbaDatabase' {
+
+    #region Mocks
+    Mock -CommandName Get-DbaDatabase -MockWith {
+        Import-Clixml -Path .\PSKoans\Koans\dbatools\Mocks\Database_All.xml
+    } -ParameterFilter { $_.SqlInstance -eq 'localhost' }
+    
+    Mock -CommandName Get-DbaDatabase -MockWith {
+        Import-Clixml -Path .\PSKoans\Koans\dbatools\Mocks\Database_TestDb.xml
+    } -ParameterFilter { $_.SqlInstance -eq 'localhost' -and $_.Database -eq 'testdb' }
+
+    Mock -CommandName Get-DbaDatabase -MockWith {
+        Import-Clixml -Path .\PSKoans\Koans\dbatools\Mocks\Database_System.xml
+    } -ParameterFilter { $_.SqlInstance -eq 'localhost' -and $_.ExcludeUser -eq $true }
+
+    Mock -CommandName Get-DbaDatabase -MockWith {
+        Import-Clixml -Path .\PSKoans\Koans\dbatools\Mocks\Database_TestDb.xml
+    } -ParameterFilter { $_.SqlInstance -eq 'localhost' -and $_.ExludeSystem -eq $true }
+    #endregion
+
+    # `Get-DbaDatabase` requires one thing; A SQL Server instance name.
+    # You can pass in "localhost" for the default name for a SQL Server instance.
+    # The simplest usage of `Get-DbaDatabase` is to run it and passing in the name
+    # of the SQL Server instance. This will get information about all the databases
+    # on the instance.
```

Comment:
Please use multiline comments with extra indents for comment blocks, like this:
```powershell
<#
    ...
    You can pass in "localhost" for the default name for a SQL Server instance.
    ...
#>

Reply:
Done! Got to practice a bit of vim substitution for this one 😄